docs/getting-started/fundamentals/static-dynamic-rendering.md

```diff
@@ -0,0 +1,27 @@
+# Static or Dynamic rendering of a block
+
+
+
+A block can define dynamic rendering can be defined in two ways:
+    - The `register_block_type()` function includes a `render_callback` argument.
+    - A `render` property is added to `block.json`, whose value points to a separate PHP file (usually named `render.php`).
```

Comment:
I feel like this is very confusing. This explains how to use dynamic rendering without explaining first what dynamic rendering even is / letting the reader know that there are two different kinds of rendering